src/components/disease/diseaseComparisonRibbon.js

```diff
@@ -128,23 +139,35 @@ class DiseaseComparisonRibbon extends Component {
         </div>
 
         <HorizontalScroll>
-          <div className='text-nowrap'>
-            <GenericRibbon
-              categories={summary.data.categories}
-              colorBy={COLOR_BY.CLASS_COUNT}
-              hideFirstSubjectLabel
-              itemClick={this.onDiseaseGroupClicked}
-              newTab={false}
-              selected={selectedBlock}
-              selectionMode={SELECTION.COLUMN}
-              subjectLabel={subject => <RibbonGeneSubjectLabel gene={subject} isFocusGene={subject.id === geneId} />}
-              subjectLabelPosition={POSITION.LEFT}
-              subjects={summary.data.subjects}
-            />
+          <div className='text-nowrap' style={{'width' : '1100px' }} >
+            {
+              summary.loading ? <LoadingSpinner /> :
+                <wc-ribbon-strips 
+                  category-all-style='1'
+                  color-by='0'
+                  data={JSON.stringify(summary.data)}
+                  group-clickable={false}
+                  group-open-new-tab={false}
+                  id='disease-ribbon'
+                  new-tab={false}
+                  selected='all'
+                  selection-mode='1'
+                  subject-base-url='/gene/'
+                  subject-open-new-tab={false}
```

Comment:
Is it no longer possible to supply our own subject label component? My concern is that now these links will (presumably, correct me if I'm wrong) be rendered with plain `<a>` elements. Those will cause a full page reload and give a bad experience for our single page application.

Reply:
That is correct, those are traditional <a> links that will cause a full page reload. This is by design as we also want those links to open new tabs for other integrations.

However, I think there are two ways to solve that:
* web components do accept slots, so it's possible to pass your own html code instead but I'll have to investigate this a bit further to see if there is no issue to inject a REACT html code
* i could possibly also create another parameter to specify that these links should not reload the page and use something like https://gomakethings.com/how-to-update-a-url-without-reloading-the-page-using-vanilla-javascript/

In both cases, this will need a bit more try, so probably fixed more at the next release as my priority will be on the GO ribbon

Reply:
The slots thing sounds interesting. I don't know much about it, but if that works, great. If that doesn't work why not just keep it simple and provide a parameter that's basically a click event handler that gets put on the `<a>` elements? Then the alliance code can `preventDefault` and do the necessary navigation work. 

Obviously the GO ribbon needs to get done, but to me this is the next highest priority thing -- higher than the other things listed in this PR.